collect_app/src/androidTest/java/org/odk/collect/android/support/rules/BlankFormTestRule.kt

```diff
@@ -1,66 +1,22 @@
 package org.odk.collect.android.support.rules
 
-import android.app.Activity
-import android.app.Application
-import android.content.Intent
-import androidx.test.core.app.ActivityScenario
-import androidx.test.core.app.ApplicationProvider
-import org.junit.rules.ExternalResource
-import org.odk.collect.android.activities.FormFillingActivity
-import org.odk.collect.android.external.FormsContract
-import org.odk.collect.android.formmanagement.FormFillingIntentFactory
-import org.odk.collect.android.injection.DaggerUtils
-import org.odk.collect.android.storage.StorageSubdirectory
-import org.odk.collect.android.support.CollectHelpers
-import org.odk.collect.android.support.StorageUtils
 import org.odk.collect.android.support.pages.FormEntryPage
-import timber.log.Timber
-import java.io.IOException
 
 class BlankFormTestRule @JvmOverloads constructor(
     private val formFilename: String,
     private val formName: String,
     private val mediaFilePaths: List<String>? = null
-) : ExternalResource() {
+) : FormEntryActivityTestRule() {
 
-    private lateinit var scenario: ActivityScenario<Activity>
+    private lateinit var formEntryPage: FormEntryPage
 
     override fun before() {
-        setUpProjectAndCopyForm()
-        scenario = ActivityScenario.launch(activityIntent)
+        super.before()
+        setUpProjectAndCopyForm(formFilename, mediaFilePaths)
+        formEntryPage = fillNewForm(formFilename, formName)
     }
 
-    override fun after() {
-        try {
-            scenario.close()
-        } catch (e: Throwable) {
-            Timber.e(Error("Error closing ActivityScenario: $e"))
-        }
+    fun fillNewForm(): FormEntryPage {
```

Comment:
This is confusing. It indicates that calling this triggers an action but in fact, it just returns `formEntryPage` and the action was already performed in `before()`. Maybe we should just expose `formEntryPage` and get rid of this function?